SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKSimulatedPlayerDeployment.cpp

```diff
@@ -505,38 +508,65 @@ FReply SSpatialGDKSimulatedPlayerDeployment::OnLaunchClicked()
 		return FReply::Handled();
 	}
 
-	if (TSharedPtr<FSpatialGDKEditor> SpatialGDKEditorSharedPtr = SpatialGDKEditorPtr.Pin()) {
-		FNotificationInfo Info(FText::FromString(TEXT("Starting simulated player deployment...")));
-		Info.bUseSuccessFailIcons = true;
-		Info.bFireAndForget = false;
+	FNotificationInfo Info(FText::FromString(TEXT("Starting simulated player deployment...")));
+	Info.bUseSuccessFailIcons = true;
+	Info.bFireAndForget = false;
 
-		TSharedPtr<SNotificationItem> NotificationItem = FSlateNotificationManager::Get().AddNotification(Info);
+	TWeakPtr<SNotificationItem> NotificationItem = FSlateNotificationManager::Get().AddNotification(Info);
 
-		NotificationItem->SetCompletionState(SNotificationItem::CS_Pending);
+	NotificationItem.Pin()->SetCompletionState(SNotificationItem::CS_Pending);
```

Comment:
Pin can fail - you should validate it before using. I realise that's very unlikely as the notification was just made, but worth doing.